src/compiler/utilitiesPublic.ts

```diff
@@ -1794,7 +1793,6 @@ export function isDeclarationBindingElement(bindingElement: BindingOrAssignmentE
     return false;
 }
 
-/** @internal */

```

Comment:
This is primarily used internally for handling destructuring assignments because we don't re-parse the cover grammar. I'm hoping I can eventually switch us to re-parsing the cover grammar instead, so making this public could be a breaking change later.

Reply:
I can unexport it if you want, sure; it was just used in our own transforms so it seemed like it would be one that could be used externally too.

Reply:
I've removed it.